packages/gatsby-plugin-utils/README.md

```diff
@@ -63,3 +63,19 @@ if (isGatsbyNodeLifecycleSupported(`createSchemaCustomization`)) {
   }
 }
 ```
+
+### `hasFeature`
+
+Feature detection is now part of Gatsby. Has a plugin author you don't know what version of Gatsby a user is using. `hasFeature` allows you to check if the current version of Gatsby has a certain feature.
+
+Here's a list of features:
+
+#### Example
+
+```js
+const { hasFeature } = require(`gatsby-plugin-utils`)
+
+if (!hasFeature(`image-service`)) {
+  polyfillImageService()
```

Comment:
This is pretty clear already given the context, but it might be useful to add a comment or change the name of the `polyfillImageService` function in the example (e.g. `yourCustomImageServicePolyfill`) so people don't get the impression this is something we export somewhere

Reply:
Well we're going to export it in the next PR but I'll write some pseudo-code instead. Good call!